src/pages/Product.js

```diff
@@ -1,21 +1,13 @@
 import React, { useState, useEffect } from "react";
-import { useParams, useHistory } from "react-router-dom";
+import { useParams } from "react-router-dom";
 import { MainNav, Footer } from "../components";
 import { getProductoByIdAsync } from "../actions/productosActions";
-import { isAuthenticated } from "../utils/isAuthenticated";
 import { PUBLIC_URL } from "../config/constants";
 import { Link } from "react-router-dom";
 
 const Product = () => {
   const [product, setProduct] = useState(null);
```

Comment:
I think we should change the name of variable, instead of product we can use "artwork" or similar